src/main/resources/assets/computercraft/lua/rom/apis/pastebin.lua

```diff
@@ -0,0 +1,146 @@
+
+--- Attempts to guess the pastebin ID from the given code or URL
+local function extractId(paste)
+  local patterns = {
+      "^([%a%d]+)$",
+      "^https?://pastebin.com/([%a%d]+)$",
+      "^pastebin.com/([%a%d]+)$",
+      "^https?://pastebin.com/raw/([%a%d]+)$",
+      "^pastebin.com/raw/([%a%d]+)$",
+  }
+
+  for i = 1, #patterns do
+      local code = paste:match( patterns[i] )
+      if code then return code end
+  end
+
+  return nil
+end
+
+function download(url)
+  if not http then
+    return false, "Pastebin requires http API"
```

Comment:
So I'd be inclined to make this return `nil, <error message>` on failure, and just `<contents>` on success (and same with other functions). This is what most other Lua and CC functions do (see `load`, `http.*`, etc...)